presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -277,6 +277,74 @@ const HISTOGRAM_PROPERTIES = {
     disableHiddenCheck: true,
 };
 
+class RuntimeStatsList extends React.Component {
```

Comment:
FYI: I guess you're aware that I started to use `functional component` instead of `class component`. And I haven't refactored the existing component code yet. I should add this to one of the to-do lists for the Presto UI. I will revisit all the class components later.

Reply:
@yhwang I wasn't aware of that. I'm not fluent in UIs, hence, was just copy-pasting from existing code. Thank you for letting me know.